main.py

```diff
@@ -322,15 +322,28 @@ class Missile(pygame.sprite.Sprite):
 
     def __init__(self, pos, missile_type):
         super(Missile, self).__init__()
-        self.image = pygame.image.load(
-            f"assets/missiles/missile-{missile_type}_fly-0.png"
-        ).convert_alpha()
+        self.images = []
+        i = 0
+        while i < 10:
+            self.images.append(
+                pygame.image.load(
+                    f"assets/missiles/missile-{missile_type}_fly-{i}.png"
+                ).convert_alpha()
+            )
+            i += 1
```

Comment:
```suggestion
        for i in range(10):
            self.images.append(
                pygame.image.load(
                    f"assets/missiles/missile-{missile_type}_fly-{i}.png"
                ).convert_alpha()
            )
```

Same effect, slightly more pythonic